app/assets/javascripts/discourse/app/templates/application.hbs

```diff
@@ -52,7 +52,13 @@
   <div id="main-outlet-wrapper" class="wrap" role="main">
     <div class="sidebar-wrapper">
       {{! empty div allows for animation }}
-      {{#if (and this.sidebarEnabled this.showSidebar)}}
+      {{#if
+        (and
+          this.sidebarEnabled
+          this.showSidebar
+          (not this.sidebarHiddenViaService)
```

Comment:
I'm not a huge fan of having so many conditions here, but I can't for the life of me figure out how to get the service state to trigger properly in `sidebarEnabled` or `showSidebar`.